doc/src/cylc-user-guide/cug.tex

```diff
@@ -1117,9 +1116,8 @@ \subsection{Locating Suite And Task Output}
 if empty when the task finishes. The suite
 \lstinline=share/= directory is made available to all tasks (by
 \lstinline=$CYLC_SUITE_SHARE_DIR=) as a common share space. The task submission
-number increments from 1 if a task retries on failure; this is used a
-sub-directory of the log tree to avoid overwriting log files from earlier
-job submissions.
+number increments from 1 if a task retries; this is used a sub-directory of the
```

Comment:
There seems to be a missing word in the original text that hasn't been picked up on: 'used **as** a sub-directory' (or similar).

Reply:
thanks, fixed.